cmd/zqd/listen/command.go

```diff
@@ -258,6 +265,27 @@ func (c *Command) initSuricata() error {
 	return nil
 }
 
+func (c *Command) initWorkers() error {
+	// This is for local testing only, at this point.
+	// Workers will be available through a K8s service in a prod deployment.
+	if c.workers != "" {
+		workers := strings.Split(c.workers, ",")
+		for i, addr := range workers {
+			// default host to 127.0.0.1
+			host, port, err := net.SplitHostPort(addr)
+			if err != nil {
+				return err
+			}
+			if host == "" {
+				host = "127.0.0.1"
+			}
+			workers[i] = "http://" + host + ":" + port
+		}
+		driver.WorkerURLs = workers
```

Comment:
nit: IMHO, this really should be a config option in zqd.Config. In general I like to avoid using globals because they give you less flexibility for writing in process unit tests.

Reply:
I agree that we should not keep the driver.WorkerURLs global. I'm not sure about using zqd.Config for this, either. I would prefer to give it some thought and change it in a later PR.

Reply:
Sure